src/commands/import.ts

```diff
@@ -743,12 +749,21 @@ command
         projectTitle,
         proxyUrl,
         repositoryMappingsPath,
+        skipCertificateVerification,
         skipUpdateCheck,
         verbose,
       } = opts;
 
       const logger = createLogger(verbose);
 
+      if (skipCertificateVerification) process.env.NODE_TLS_REJECT_UNAUTHORIZED = '0';
+
+      if (proxyUrl && !skipCertificateVerification) {
```

Comment:
This maintains compatibility with the existing approach of setting the environment variable:

```suggestion
      if (proxyUrl && process.env.NODE_TLS_REJECT_UNAUTHORIZED !== '0') {
```